Containerfile

```diff
@@ -2,7 +2,8 @@ FROM quay.io/forem/ruby:2.7.2 as builder
 
 USER root
 
-RUN curl -sL https://dl.yarnpkg.com/rpm/yarn.repo -o /etc/yum.repos.d/yarn.repo && \
+RUN curl -sL https://rpm.nodesource.com/setup_lts.x && \
```

Comment:
Why are we adding this repo in when the main Fedora repo provides Node?

Reply:
Ah, I extracted this code from a different PR and I wasn't aware that it's redundant. Removing!

Reply:
@jdoss Does Fedora only include LTS versions? If yes, this is indeed not needed.

Reply:
IIRC they track LTS. Regardless, we should just follow whatever node version that is currently stable in Fedora as a forcing function to upgrade.